src/lib/holocene/code-block.svelte

```diff
@@ -1,15 +1,35 @@
 <script lang="ts">
+  import type { HTMLAttributes } from 'svelte/elements';
+
   import Icon from '$lib/holocene/icon/icon.svelte';
   import { copyToClipboard } from '$lib/utilities/copy-to-clipboard';
   import {
     parseWithBigInt,
     stringifyWithBigInt,
   } from '$lib/utilities/parse-with-big-int';
 
+  type BaseProps = HTMLAttributes<HTMLDivElement> & {
+    content: Parameters<typeof JSON.stringify>[0];
+    inline?: boolean;
+    language?: string;
+    async?: boolean;
+    testId?: string;
+  };
+
+  type CopyableProps = BaseProps & {
```

Comment:
This typing doesn't appear to be working as expected (I think because we're defaulting `copyable` to `true`). Any ideas how we can ensure `copyIconTitle` and `copySuccessIconTitle` are included if `copyable` is not passed as a prop?

Reply:
Hm ok, I can check this out locally